java/jMakeOdf/src/make_odf/Pipe.java

```diff
@@ -299,8 +283,9 @@ public void writeAdditionalReleases(PrintWriter outfile, String pipeNr) {
 				Release release = releases.get(k);
 				String releaseName = pipeNr + "Release"
 						+ NumberUtil.format(k + 1);
-				outfile.println(releaseName + "=." + File.separator
-						+ release.fileName);
+				String fullLine = releaseName + "=." + File.separator
```

Comment:
Is there any reason, why we need the useless ".\" in front of all file names?

I do not intent to support this in the new package format.